app/src/main/java/mozilla/lockbox/view/DisplayItemFragment.kt

```diff
@@ -192,14 +192,7 @@ class DisplayItemFragment : BackableFragment(), DisplayItemView {
     }
 
     override fun showToastNotification(@StringRes strId: Int) {
-        assertOnUiThread()
-        val toast = Toast(activity)
-        toast.duration = Toast.LENGTH_SHORT
-        toast.view = layoutInflater.inflate(R.layout.toast_view, this.view as ViewGroup, false)
-        toast.setGravity(Gravity.FILL_HORIZONTAL or Gravity.BOTTOM, 0, 0)
-        val v = toast.view.findViewById(R.id.message) as TextView
-        v.text = resources.getString(strId)
-        toast.show()
+        (activity as RootActivity).showToastNotification(strId = strId, viewGroup = this.view as ViewGroup)
```

Comment:
This opens up another API to do the same thing. That will make things harder to change/copy/use in the future.

This should be achieved with sending a `ToastNotification` action from the presenter. Again, getting rid of the `ViewGroup` requirement makes this easier.